nevergrad/parametrization/choice.py

```diff
@@ -158,7 +158,8 @@ def mutate(self) -> None:
 
 
 class TransitionChoice(BaseChoice):
-    """Ordered categorical parameter, choosing one of the provided choice options as a value, with continuous transitions.
+    """Categorical parameter, choosing one of the provided choice options as a value, with continuous transitions.
+    By default, this is ordered, and most algorithms except OnePlusOne based algorithms will consider it as ordered.
```

Comment:
Isn't it "DiscreteOnePlusOne" rather than OnePlusOne ? I would expect OnePlusOne to keep the ordered style.